src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,55 +1,110 @@
-import React from 'react';
+import React, {
+  FC, useState, ChangeEvent, FormEvent, /* KeyboardEvent */
+} from 'react';
 import './FindMovie.scss';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+const URL = 'https://www.omdbapi.com/?apikey=5c8f0f93&t=';
+const URL_IMDB = 'https://www.imdb.com/title/';
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
-        </div>
+interface Props {
+  addMovie(movie: Movie): void;
+}
+
+export const FindMovie: FC<Props> = ({ addMovie }) => {
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [isError, setIsError] = useState(false);
+
+  function handleInput(event: ChangeEvent<HTMLInputElement>) {
+    setQuery(event.target.value);
+    setIsError(false);
+  }
 
-        <div className="control">
-          <button
-            type="button"
-            className="button is-primary"
-          >
-            Add to the list
-          </button>
+  function submitHandler(event: FormEvent<HTMLFormElement>) {
+    event.preventDefault();
+    fetch(URL + query)
+      .then(movieFromApi => {
+        return movieFromApi.json();
+      })
+      .then(movieApi => {
+        const {
+          Title: title,
+          Plot: description,
+          Poster: imgUrl,
+          imdbID: imdbId,
+          Response,
+        } = movieApi;
+
+        if (Response === 'True') {
+          const imdbUrl = URL_IMDB + imdbId;
```

Comment:
не то чтобы очевидно, строки конкатинируешь или математические вычисления производишь
```suggestion
          const imdbUrl = `${URL_IMDB}${imdbId}`;
```